api/src/libs/migrateBusinessesFromUsers.test.ts

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
[dust] I'd prefer if we didn't have to do these. Are they still needed for this?

Reply:
Yes, the suppression (the `any`) is needed. We're disabling @typescript-eslint/no-explicit-any because we only need a partial mock of the Business object with the relevant properties for testing. Implementing all members would unnecessarily complicate the mock. Using any keeps the mock focused and simple without requiring all properties, which isn't needed for the test.